test/apps/rails2/app/views/home/test_model.html.erb

```diff
@@ -11,3 +11,5 @@ It's just a model <%= link_to "Hipster ipsum", User.first %>
 It's just a couple of models <%= link_to "Hipster ipsum", [Account.first, User.last] %>
 
 <%= render :partial => 'models', :collection => all_the_things, :as => :model %>
+
+Bad for 2.x: <%= link_to "test", params[:user_id].u %>
```

Comment:
While this happens to work... `u` would be actually called `u(params[:user_id])` :hamster: